test/integration/keep-alive.js

```diff
@@ -62,7 +62,7 @@ test('Client not using keepalive does not detect disconnect - separated by proce
 
   const clientTest = t.test('client')
   const { bootstrap } = await swarm(t)
-  const node = new DHT({ bootstrap })
+  const node = new DHT({ bootstrap, connectionKeepAlive: 0 })
```

Comment:
This test tests the case where no keepAlive is set (previously default, now explicit). We might just remove this test now, as it's not that relevant anymore